providers/providers_and_managers.md

```diff
@@ -0,0 +1,34 @@
+# What are Providers and Managers?
+
+The terms Provider, Manager, and EMS are all used to some degree interchangeably within the ManageIQ Community however they do have more specific meanings.
+
+### ExtManagementSystem / Manager
+
+Managers represent a grouping of resources by functional classification, e.g.: Cloud, Network, Storage.  For example the `CloudManager` has `vms` and `images`, the `NetworkManager` has `cloud_networks` and `security_groups`, and the `StorageManager` has `cloud_volumes` and `cloud_volume_snapshots`.  This allows for better grouping of related resources.
+
+The managers represent the different "flavors" of inventory, if a completely new type of manager were brought to ManageIQ we would add a new manager class to represent it.
```

Comment:
Let's go with the "family" terminology (also, I think this line can be combined with the one on line 7 into a single paragraph).

Reply:
:+1: Done